mkdocs/utils/__init__.py

```diff
@@ -321,7 +321,7 @@ def path_to_url(path):
     if os.path.sep == '/':
         return path
 
-    return pathname2url(path)
+    return pathname2url(path.encode('utf8'))
```

Comment:
So, given the failures maybe we should only do the encode on windows and Python 3?